Makefile

```diff
@@ -17,3 +17,14 @@ local-dev-shell:
 local-dev-down:
 	docker stop local-dev
 	docker rm local-dev
+
+local-prod-build:
+	docker build -t local-prod-image -f ./docker/prod/Dockerfile .
+local-prod-run:
+	docker run -d --name local-prod local-prod-image
+	docker exec -it local-prod /bin/bash
+local-prod-shell:
```

Comment:
In a future PR can you include some readme comments explaining the purpose of each command and when they should be used